var/spack/repos/builtin/packages/acts/package.py

```diff
@@ -226,6 +231,7 @@ class Acts(CMakePackage, CudaPackage):
         when="@0.16:",
     )
     variant("fatras_geant4", default=False, description="Build Geant4 Fatras package")
+    variant("geomodel", default=False, description="Build GeoModel plugin", when="@33:")
```

Comment:
How is this variant supposed to be used in the build?

Reply:
Ah, oops, forgot to add the cmake_args entry. Thanks!

Reply:
49836fe1356bd3fb21b8d599d784c38bcfe40018